lib/puppet/face/strings.rb

```diff
@@ -50,6 +50,7 @@ def check_required_features
       # For now, assume the remaining positional args are a list of manifest
       # and ruby files to parse.
       yard_args = (args.empty? ? MODULE_SOURCEFILES : args)
+      class YARD::Logger; def progress(*args); end; end
```

Comment:
@iankronquist Could you go ahead and make this change? Just comment it out and leave a comment above explaining why it is useful for debugging (e.g. prevent threading so we can use the debugger)